docs/core/sdk/file-based-apps.md

```diff
@@ -29,7 +29,7 @@ Adds a NuGet package reference to your application.
 
 ```csharp
 #:package Newtonsoft.Json
```

Comment:
Note that not specifying a version only works with central package management (Directory.Packages.props), so not sure if you want to have that here, perhaps better would be:
```suggestion
#:package Newtonsoft.Json@*
```